pkg/install/stack/usercluster-mla/validation.go

```diff
@@ -71,6 +71,27 @@ func validateHelmValues(helmValues *yamled.Document, opt stack.DeployOptions) []
 		}
 	}
 
+	if helmValues.Has(yamled.Path{"cortex", "config", "ruler", "storage"}) {
+		failures = append(failures, fmt.Errorf("cortex.config.ruler.storage is not supported, please use cortex.config.ruler_storage instead"))
```

Comment:
My only nit about this error message would be.. to use `no longer supported` message instead of `not supported` like the message on line 91.

Same suggestion for line 78 as well.

Reply:
thanks, nice catch!